sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +72,74 @@ def setup(self):
             "version"
         ]
 
-        self.add_cmd_output([
-            f"ceph daemon {m} {cmd}" for m in self.get_socks() for cmd in cmds]
-        )
-
-    def get_socks(self):
+        if not microceph_pkg:
```

Comment:
One minor nit: for easy future extensions of the supported packages (like microceph), something like this should be preferred:
```
if microceph_pkg:
    # microceph related config
elif x_pkg:
    # x related config
else:
    # standard/default config

Reply:
Hi Utkarsh,

this logically makes sense, but as of now there's only one variant of ceph i.e microceph for which sos must account for. I'd rather keep the structure of all plugins consistent for now, so will implement this structure when its required, i.e when there is a third alternative. Does this sound OK?